soc/nordic/nrf54h/pinctrl.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/drivers/pinctrl/pinctrl_nrf_common.h>
+#include <nrf/gpd.h>
+
+static bool gpd_requested;
+
+int nrf_pinctrl_hook_start(const pinctrl_soc_pin_t *pins, uint8_t pin_cnt,
+			   uintptr_t reg)
+{
+	gpd_requested = false;
+
+	return 0;
+}
+
+bool nrf_pinctrl_hook_custom_periph(const struct nrf_pinctrl_pin *pin,
+				    struct nrf_pinctrl_config *cfg)
+{
+	return false;
+}
```

Comment:
How about having separate Kconfig options for particular hooks, so that such things could be avoided?

Reply:
No strong feelings about this - for me it's just more readable to add empty C functions than more Kconfigs.